guides/release/accessibility/application-considerations.md

```diff
@@ -1,15 +1,31 @@
 In this section, you will learn about the configurations and libraries that affect an entire application's accessibility.
 
-## Language attribute
+## Lang Attribute
 
-Every application must have a primary language declaration. This language declaration is important for assistive technology like screen readers, internationalization tools built into browsers, and search engines.
+Declaring the language of the HTML document explicitly allows users to be better able to understand your content.
 
-To indicate the primary language, use the `lang` attribute on the `<html>` element. Until Ember provides this by default, developers should edit the `index.html` file of the generated Ember application with the default language declaration. This is inherited by all other elements, and will set a default language for the text in the document head element.
+> Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. As a result, users with disabilities will be better able to understand the content.
+> [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
 
-If there happens to be any content on the page that is in a different language from that declared in the `<html>` element, the `lang` attribute can be used on the parent element to indicate a different language.
+A primary language should be defined on the `<html>` element's `lang` attribute. For existing Ember apps a developer may edit the `index.html` file or leverage [ember-intl](https://github.com/ember-intl/ember-intl). [Soon Ember CLI will allow for creating new applications with this attribute set for you automatically](https://github.com/ember-cli/ember-cli/pull/9259).
```

Comment:
Although [RFC #635](https://github.com/emberjs/rfcs/pull/635) has been accepted, since the work is in progress, I don't think it's a good idea to mention in the Guides a feature that doesn't exist yet.

I think a better approach may be to add a TODO item in https://github.com/ember-cli/ember-cli/pull/9259. Once the `--lang` feature lands, we update the Guides once more (an alternative location may be Ember CLI Guides).

Would this be okay with you?

```suggestion
A primary language should be defined on the `<html>` element's `lang` attribute. For existing Ember apps, a developer may edit the `index.html` file or leverage [ember-intl](https://github.com/ember-intl/ember-intl).
```

Reply:
Sounds great, mentioned in the ember-cli PR this as a future TODO item.